jersey-tck/src/test/resources/arquillian.xml

```diff
@@ -0,0 +1,15 @@
+
+<arquillian>
+<container qualifier="arq-glassfish-managed">
+    <configuration>
+        <property name="allowConnectingToRunningServer">false</property>
+        <!--            <property name="glassFishHome">target/${glassfish.home}</property>-->
+        <property name="adminHost">localhost</property>
+        <property name="webServerHost">localhost</property>
+        <property name="webServerPort">8080</property>
+        <property name="adminPort">4848</property>
+        <property name="enableDerby">${enableDerby:true}</property>
```

Comment:
Why is Derby needed?

Reply:
Removed the file as it is not used with the current changes.